clients/fides-js/rollup.config.mjs

```diff
@@ -24,8 +24,8 @@ const GZIP_SIZE_HEADLESS_ERROR_KB = 25;
 const GZIP_SIZE_HEADLESS_WARN_KB = 20;
 
 // GPP
-const GZIP_SIZE_GPP_ERROR_KB = 25;
-const GZIP_SIZE_GPP_WARN_KB = 15;
+const GZIP_SIZE_GPP_ERROR_KB = 40;
```

Comment:
Confirming this size bump via client-services... stay tuned

Reply:
any lingering concern here, or are we ok with the increase?